src/test/java/tests/TestSalesfoce.java

```
@@ -0,0 +1,49 @@
+package tests;
+
+import api.SalesforceApi;
+import org.apache.http.HttpResponse;
+import org.testng.Assert;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+import pageObjects.BasePage;
+import pageObjects.LoginPage;
+
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+
+public class TestSalesfoce extends BasePage {
+
+    private String journyStr = "";
+
+
+    @BeforeTest
+    public void before() {
+        /* Generating new lead */
+        try {
+            SalesforceApi salesforceApi = new SalesforceApi();
+            HttpResponse response = null;
+            response = salesforceApi.generateLead();
+            Assert.assertTrue(response.toString().contains("200 OK"));
+            journyStr = salesforceApi.customerStr();
+            System.out.println("Journey created successfully: " + journyStr);
+        } catch (Exception e) {
+            e.printStackTrace();
```

if it fails the test should fail, no? why are you catching and ignoring?

---

agree, even though the test will fail once try to search for string that was not created.
i do agree with you that test will fail faster on creation of lead instead of waiting for it to search for values